README.md

```diff
@@ -174,3 +174,14 @@ Azure Blobs doesn't support multiple writers for the same blob.  With the way `B
 ## Changes
 
 See [CHANGES](CHANGES.md)
+
+## Contributing
+
+Get your account added to the appropriate [testing buckets](https://github.com/christopher-hesse/blobfile/blob/bb885e72e97ddade675d4493db235b43888a2191/blobfile/_ops_test.py#L30-L36)
+
+```sh
+conda env create -f env.yaml -n blobfile
+conda activate blobfile
+# run tests
+python testing/run-tests.py -vs

```

Comment:
If people are running this script directly, I think `run-tests.py` and `run-static.py` should be combined into a single file.

Reply:
Might be worth moving the contents of `testing/README.md` here and deleting that file so it's all in one place.

Reply:
moved